ocp_utilities/monitoring.py

```diff
@@ -129,41 +129,45 @@ def get_all_alerts_by_alert_name(self, alert_name):
                 alert_list.append(alert)
         return alert_list
 
-    def get_firing_alerts(self, alert_name):
```

Comment:
You renamed the function, this is a breaking change
Please make sure to mention it in the pr title